extensions/sd_api_pictures/script.py

```diff
@@ -43,23 +43,23 @@ def give_VRAM_priority(actor):
     if actor == 'SD':
         unload_model()
         print("Requesting Auto1111 to re-load last checkpoint used...")
-        response = requests.post(url=f'{params["address"]}/sdapi/v1/reload-checkpoint', json='')
+        response = requests.post(url=f'{params["address"]}/sdapi/v1/reload-checkpoint', json='', verify=shared.args.disable_ssl_verify)
```

Comment:
I don't like adding global flags that only apply to the sd extension. Those should be removed and only the gradio flags should be kept. 

The sd flags could instead be turned into booleans into the params dict for the extension, making them customizable through settings.json